packages/terra-avatar/docs/upgrade-guide.md

```diff
@@ -0,0 +1,37 @@
+# terra-avatar Upgrade Guide
+## V2 Major Updates
+### Removed
+* `variant` prop. Avatar is now split into three sub components - Avatar, Facility, and SharedUser. Import them by using named imports.
+
+### Added
+* `alt` prop. Represents alternative text. This is required on all variants for accessibility.
+* `color` prop. This controls theme specific colors. By default, it is set to `auto`. `auto` hashes the `alt` prop to generate a color. See prop tables for accepted values.
+* `hashValue` prop. This value will be used in place of `alt` to generate a color. Use this if `alt` should not be used to generate a color.
```

Comment:
Thank you. I was about to suggest this to keep feature parity for CareAware Connect but you're already on it.